lib/tasks/journeys_report.rake

```diff
@@ -64,11 +89,10 @@ namespace :journeys do
         file.write(JSON.pretty_generate({ moves: moves })) # maybe change to .to_json for production
       end
 
-      puts "Exporting #{filename} to S3..."
-      bucket_name = ENV['S3_REPORTING_BUCKET_NAME']
       if bucket_name.present?
+        puts "Exporting #{filename} to S3..."
         s3 = Aws::S3::Resource.new
-        obj = s3.bucket(bucket_name).object(filename)
+        obj = s3.bucket(bucket_name).object("#{timestamp}/#{filename}")
```

Comment:
Using the `timestamp`as a prefix (ends up looking like a directory).
Kept the timestamp in the file regardless, as it it convenient when you pull a flat list of the keys (striping all the prefixes).